libs/core/core/components/use-back-forward-button-preventer.hook.tsx

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { useCallback, useEffect } from "react"
+
+const useBackForwardButtonPreventer = () => {
+  const handleMouseUp = useCallback((event: MouseEvent) => {
+    if (event.button === 3 || event.button === 4) {
```

Comment:
Are you sure those values will always catch the prev & next buttons? What about gaming mouses etc.?

Reply:
Values 3 and 4 are standard and typically handle the 'back' and 'forward' buttons in most cases, according to the MDN documentation. Even with gaming mouses, which might allow remapping, standard browsers generally still recognize these buttons according to this standard. More details can be found in the MDN documentation: [MouseEvent.button](https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/button). 

I don't have a gaming mouse, so I'm basing my response on the documentation and information available online. If you have such a mouse, I can conduct a test to verify this.

Reply:
Ok, that should be enough 👍